lib/twitter-ads/version.rb

```diff
@@ -2,5 +2,5 @@
 # Copyright (C) 2015 Twitter, Inc.
 
 module TwitterAds
-  VERSION = '1.0.0'.freeze
+  VERSION = '1.1.0'.freeze
```

Comment:
I don't know why this got picked up as a change, it definitely should already be 1.1 ... (need to squash my commits I guess)